manifests/install.pp

```diff
@@ -117,7 +117,7 @@
   } elsif ($::osfamily == 'RedHat') and ($package_version != 'present') {
     # Workaround PUP-5802/PUP-5025
     if ($::operatingsystem == 'Fedora') {
-      $pkg_os_suffix = 'fedoraf'
+      $pkg_os_suffix = 'fedora'
```

Comment:
Just Puppet 5 (and beyond). Supporting both names would be good. I don't know a lot about this repo, so wanted to get this PR up for feedback like this. 😄